src/Components/ADT3DSceneBuilder/Internal/Behaviors/Widgets/WidgetBuilders/DataHistoryWidgetBuilder/DataHistoryWidgetBuilder.tsx

```diff
@@ -201,13 +218,11 @@ const DataHistoryWidgetBuilder: React.FC<IDataHistoryWidgetBuilderProps> = ({
                     <TooltipCallout
                         content={{
                             buttonAriaLabel: t(
-                                'widgets.dataHistory.form.connectionStringInformation'
+                                LOC_KEYS.connectionStringInformation
                             ),
                             calloutContent: (
                                 <>
-                                    {t(
-                                        'widgets.dataHistory.form.connectionStringInformation'
-                                    )}{' '}
+                                    {t(LOC_KEYS.connectionStringInformation)}{' '}
                                     <Link
                                         target="_blank"
                                         href={DOCUMENTATION_LINKS.dataHistory}
```

Comment:
@msnyder-msft can you generate another go.microsoft link for this one and send me please?
https://learn.microsoft.com/en-us/azure/digital-twins/concepts-data-history